GSL1680.cpp

```diff
@@ -77,20 +90,14 @@ void GSL1680::clear_reg()
     uint8_t DATA[4] = {0x88, 0x01, 0x04, 0x00};
     uint8_t TIMER[4] = {20, 5, 5, 20};
 
-    Wire.beginTransmission(I2CADDR);
+    uint8_t _data[1];
 
     int i;
     for (i = 0; i < 4; ++i) {
-        Wire.write(REG[i]);
-        Wire.write(DATA[i]);
-        delay(TIMER[i]);
```

Comment:
No need for a delay here? I don't have the hardware anymore but I remember spending a lot of time figuring out the timings, it wouldn't work otherwise